opennms-config-model/src/main/java/org/opennms/netmgt/config/discovery/Definition.java

```diff
@@ -101,6 +101,13 @@
     @XmlElement(name = "include-url")
     private List<IncludeUrl> includeUrls = new ArrayList<>();
 
+    /**
+     * a file URL holding specific addresses to be
+     *  polled
```

Comment:
`a file URL holding specific addresses to be excluded`

Reply:
-Added the proper comments
-Added include and exclude url to equals and hashCode methods